app/api/campaigns/index.js

```diff
@@ -70,6 +70,20 @@ router.get('/:id/submissions', async (req, res) => {
   }
 });
 
+router.get('/skills/:skill', async (req, res) => {
+  const { skill } = req.params;
```

Comment:
Can we validate this? First convert it to CONST_CASE (`skill.toUpperCase().replace(' ','_')`), then throw a 400 if that skill is not found in the enum.